src/main/features/core/tray.js

```diff
@@ -40,9 +63,35 @@ const setContextMenu = () => {
 setContextMenu();
 
 appIcon.setToolTip('Google Play Music');
-appIcon.on('double-click', () => {
-  mainWindow.setSkipTaskbar(false);
-  mainWindow.show();
+
+
+// Iconified     -> show, to foreground
+// In background -> to foreground
+// In foreground -> iconify
+appIcon.on('click', () => {
```

Comment:
Can we extract this to it's own function `toggleAppVisiblitiy` or something similar?

Then have the same action on single click and double click?